GVFS/GVFS.Upgrader/UpgradeOrchestrator.cs

```diff
@@ -219,6 +219,25 @@ private bool TryRunUpgrade(out Version newVersion, out string consoleError)
                 return false;
             }
 
+            if (!this.LaunchInsideSpinner(
+                () =>
+                {
+                    if (!this.preRunChecker.IsInstallationBlockedByRunningProcess(out error))
+                    {
+                        return false;
+                    }
+
+                    this.mount = true;

```

Comment:
Why is `this.mount` being set to `true`?  Won't it already be set to `true` on line 211?